www/lynx/Portfile

```diff
@@ -3,9 +3,9 @@
 PortSystem          1.0
 
 name                lynx
-set milestone       2.8.9
-version             ${milestone}rel.1
-revision            2
+set milestone       2.9.0
+version             ${milestone}
+revision            0
 categories          www
 platforms           darwin
 license             {GPL-2 OpenSSLException}
```

Comment:
presumably this will now be using OpenSSL v3.x - so doesn't that eliminate the need for the OpenSSLException as the license for v2 is less restrictive than for v2?

Reply:
Actually the exception deals mainly with the politics of Debian, and probably no one else cares about the distinction.